src/Sarif.Sarifer/SariferExtensionOptionPage.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.CodeAnalysis.Sarif.Sarifer
+{
+    [Guid("BB3665D5-E661-48C0-801A-19B034F3CD5F")]
+    public class SariferExtensionOptionPage : DialogPage
+    {
+        private const string CategoryName = "Sarifer Options";
+        private const string DisplayName_BackgroundAnalysisEnabled = "Enable Background Analysis";
+        private const string Description_BackgroundAnalysisEnabled = "If enable background analyzer to analyze content in editor while editing.";
+        private const string DisplayName_AnalyzeSarifFile = "If Analyze .sarif File";
```

Comment:
>If Analyze .sarif File [](start = 61, length = 22)

Change to: "Enable .sarif files by default"

Reply:
done

---
In reply to: [564949549](https://github.com/microsoft/sarif-visualstudio-extension/pull/354#discussion_r564949549) [](ancestors = 564949549)

Reply:
Sorry, lets change to "Enable automatic analysis for .sarif files"
and  for the description: "If enabled, the background analysis would trigger if a .sarif file is opened or edited"

---
In reply to: [564964057](https://github.com/microsoft/sarif-visualstudio-extension/pull/354#discussion_r564964057) [](ancestors = 564964057,564949549)

Reply:
as discussed name changed to "Enable analysis for .sarif files", description not changed. 
since this option will affect all 5 analysis scenarios not only automatic analysis.

---
In reply to: [564967112](https://github.com/microsoft/sarif-visualstudio-extension/pull/354#discussion_r564967112) [](ancestors = 564967112,564964057,564949549)